src/app/shared/player/player.component.ts

```diff
@@ -94,12 +94,16 @@ export class PlayerComponent implements OnInit, OnChanges {
       }
     });
 
-    this.player.addEventListener('playing', (e: Event) => {
-      this.play.emit(e);
+    this.player.addEventListener('play', (e: Event) => {
```

Comment:
I switched from the `playing` event to the `play` event in order to fix a separate issue (unrelated to scrubbing) where the overlay wasn't being hidden upon resuming play in the PRX version of the player. Whereas RadioPublic's underlying audio player emits both the `playing` and `play` events when the play button is pressed to resume playback, the PRX dovetail player only emits the`play` event.

Reply:
Hmm... in which situation is the DovetailPlayer only emitting `play`?  I'm seeing both events.

Reply:
[Here's a branch](https://github.com/BinaryNate/play.prx.org/commit/cf2545fcbfa53bdbc9e97d0836efd07057c24840) to demonstrate the issue by listening for both 'play' and 'playing' events. The gif below shows how to reproduce the issue by starting to play a podcast, pausing it, and pressing play again. Since the 'playing' event handler is never called, `'playing event emitted'` isn't logged to the console and the overlay isn't dismissed.

![bug-demo](https://user-images.githubusercontent.com/4952192/31411227-f55bf688-adde-11e7-9aa4-0ac6ddc33cd9.gif)

Reply:
Gotcha - thanks for the explanation!